pytket/docs/circuit.rst

```diff
@@ -1,8 +1,8 @@
 pytket.circuit
 ==================================
 .. toctree::
-   :caption: Classes:
-   :maxdepth: 2
+   :caption: Submodules:
+   :maxdepth: 1
```

Comment:
seeting the depth to 1 ensures that [this page](https://tket.quantinuum.com/api-docs/circuit.html) isn't cluttered with all of the classes.

All of the contents can now be found in the right sidebar of the docs instead

Reply:
Changed it back to `Classes` for now